erpnext/patches.txt

```diff
@@ -307,6 +307,7 @@ erpnext.patches.v13_0.job_card_status_on_hold
 erpnext.patches.v14_0.copy_is_subcontracted_value_to_is_old_subcontracting_flow
 erpnext.patches.v14_0.migrate_gl_to_payment_ledger
 erpnext.patches.v14_0.crm_ux_cleanup
+erpnext.patches.v14_0.migrate_existing_lead_notes_as_per_the_new_format #1
```

Comment:
Can be ignored, but since we are doing this for Prospect as well:

```suggestion
erpnext.patches.v14_0.migrate_existing_crm_notes_as_per_the_new_format
```